core/src/main/resources/lib/hudson/propertyTable.jelly

```diff
@@ -63,14 +66,20 @@ THE SOFTWARE.
           <td style="white-space: normal" class="property-table__value">
             <j:choose>
               <j:when test="${attrs.sensitive}">
-                <div class="hidden-info">
-                  <div class="jenkins-button">
-                    ${%reveal}
+                <div class="app-hidden-info-reveal">
+                  <div class="jenkins-button jenkins-button--transparent">
+                    <span class="jenkins-!-color-blue"><l:icon src="symbol-view"/> ${%reveal}</span>
```

Comment:
It appears that the span is at fault for the misaligned symbol. Removign it aligns the button with the symbol fine again

Reply:
Will look but annoying as I couldn’t set it on the div